Campaigns/Ransomware hits healthcare - Alternate Data Streams use.txt

```diff
@@ -0,0 +1,10 @@
+//Find use of Alternate Data Streams (ADS) for anti-forensic purposes.
+// Alternate Data Streams execution 

```

Comment:
The comment is:
//Find use of Alternate Data Streams (ADS) for anti-forensic purposes.
// Alternate Data Streams execution 

Maybe only the first line is enough?

Reply:
Oh I see now, the second comment line is referring to the first line of query text and not explaining the entire query. It wasn't clear to me on first read but probably not critical to change this.